README.md

```diff
@@ -174,6 +172,39 @@ tox -e python3.9 -- tests/unit/extras/test_envfile.py::test_cannot_determine_for
 
 In short, tox passes all arguments to the right of `--` directly to pytest.
 
+## Package managers
+
+Supported so far:
+
+* [gomod](#gomod)
+
+### [gomod](https://go.dev/ref/mod)
+
+Current version: 1.18 [^go-version] [^go-compat]
+
+The gomod package manager works by parsing the [go.mod](https://go.dev/ref/mod#go-mod-file) file present in the source
+repository to determine which dependencies to download. Cachi2 does not parse this file on its own - rather, we rely on
+the `go` command to download and list the required dependencies.
+
+From go 1.17 onward, the go.mod file includes all the transitively required dependencies of your application - see the
+section about *Pruned module graphs* in the [1.17 changelog][go117-changelog]. In previous go versions, the go.mod file
+included only direct dependencies. Cachi2 does support downloading and listing all transitive dependencies for earlier
+versions thanks to Go's backwards compatibility[^go-compat]. Note that using go >= 1.17 in your project has the added
+benefit of downloading fewer dependencies (as noted in the changelog), in some cases drastically so.
+
+See [docs/gomod.md](docs/gomod.md) for more details.
+
+[^go-version]: Cachi2 expects to use a specific version of the `go` command when downloading dependencies. This is the
+  version installed in the [cachi2 container](#container-image). We do not guarantee correctness if you run Cachi2
+  locally (outside the container) with a different Go version. You *are* free to use a different version to build your
+  project.
```

Comment:
I realized we have a slight problem. If we let users run cachi2 with their own Go version, we can't guarantee correctness. For now, we should probably stick to providing only a container image.

Reply:
Should we add a (disableable) check for the system's Go version?

Reply:
For local development that might be more of a nuisance than helpful. I think Cachi2 will sort of work with any go version, though it might report incorrect metadata or not support new features

If we ever distribute it as a standalone CLI, then we probably should. Worth filing a ticket already?

Reply:
Not sure... it'll probably end up forgotten in the depths of Jira. At least, this is well documented now, I feel like it should be enough.